tidy-data/01-Reshape-Data/01-Reshape-Data.Rmd

```diff
@@ -269,46 +269,50 @@ question("What are the variables in cases?",
 
 ![](https://vimeo.com/229581273)
 
-### gather()
+### pivot_longer()
 
-You can use the `gather()` function in the **tidyr** package to convert wide data to long data. Notice that `gather()` returns a tidy copy of the dataset, but does not alter the original dataset. If you wish to use this copy later, you'll need to save it somewhere.
+You can use the `pivot_longer()` function in the **tidyr** package to convert wide data to long data. Notice that `pivot_longer()` returns a tidy copy of the dataset, but does not alter the original dataset. If you wish to use this copy later, you'll need to save it somewhere.
 
 ```{r echo = TRUE}
-cases %>% gather(key = "year", value = "n", 2, 3, 4)
+cases %>% 
+  pivot_longer(
+    cols = -Country, 
+    names_to = "year", 
+    values_to = "n"
+    )
 ```
 
-Let's take a closer look at the `gather()` syntax.
+Let's take a closer look at the `pivot_longer()` syntax.
 
-### gather() syntax
+### pivot_longer() syntax
 
 Here's the same call written without the pipe operator, which makes the syntax easier to see.
 
 ```{r echo = TRUE, eval = FALSE}
-gather(cases, key = "year", value = "n", 2, 3, 4)
+pivot_longer(cases, cols = c(`2011`, `2012`, `2013`), names_to = "year", values_to = "n")
 ```
 
-To use `gather()`, pass it the name of a data set to reshape followed by two new column names to use. Each name should be a character string surrounded by quotes: 
+To use `pivot_longer()`, pass it the name of a data set to reshape followed by which columns to pivot longer, the name of a new variable that will contain on the names of these columns as values, and the name of another new variables that will contain the values from these columns:
 
-* the key string will become the name of a new column that contains former column names. 
-* the value string will become the name of a new column that contains former cell values.
+* the `cols` argument contains the name of the columns to pivot into longer format.
+* the `names_to` argument is a string specifying the name of the column to create from the data stored in the column names of the dataset to be reshaped.
```

Comment:
May be redundant, but I like "string specifying the name of the *new* column to create..."

Reply:
👍